<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmEditor</class>
 <widget class="QDialog" name="AlarmEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix1/icons/alarm-clock.png</normaloff>:/new/prefix1/icons/alarm-clock.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxTime">
       <property name="title">
        <string>Time</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutHour">
          <item>
           <widget class="QSpinBox" name="spinBoxHour">
            <property name="minimumSize">
             <size>
              <width>39</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>23</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxMinutes">
            <property name="minimumSize">
             <size>
              <width>39</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Repeat</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutRepetition"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QTextEdit" name="textEditNote">
       <property name="minimumSize">
        <size>
         <width>313</width>
         <height>91</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="soundsList">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
       <item>
        <property name="text">
         <string>Alarm1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alarm2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alarm3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alarm4</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/icons/play-button.png</normaloff>:/new/prefix1/icons/play-button.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSoundOptions">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>24</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sound options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="playSoundLabellabel">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Play sound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="playSoundComboBox">
       <property name="minimumSize">
        <size>
         <width>164</width>
         <height>22</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Until alarm message is closed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 minute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 hour</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="turnVolumeCheckBox">
       <property name="text">
        <string>Turn on the volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="soundVolumeSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="soundValueLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="snoozeCheckBox">
       <property name="text">
        <string>Snooze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="snoozeComboBox">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>24</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1 minute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 minutes</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnsLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
